exercises/concept/lasagna-master/.meta/config.json

```
@@ -0,0 +1,12 @@
+{
+  "blurb": "Dive deeper into Go functions while preparing to cook the perfect lasagna.",
+  "authors": ["bobtfish"],
+  "contributors": [],
+  "files": {
+    "solution": ["lasagna-master.go"],
+    "test": ["lasagna-master_test.go"],
+    "exemplar": [".meta/exemplar.go"]
+  },
+  "source": "Inspired by the Lasagna Master exercise in the Javascript track",
+  "source_url": "https://github.com/exercism/javascript/blob/main/exercises/concept/lasagna-master/.docs/instructions.md"
```

You don't need the source if you specify the forked_from field (which is appropriate here)

```suggestion
  "forked_from": ["javascript/lasagna-master"]
```